### ISSUE TITLE
Crownfall Coins Missing

### ISSUE THREAD

**Issue by ToomatooSauce** [ValveSoftware/Dota2-Gameplay#20228]
### Description

I have counted the total coins that I got and noticed that I was missing 10 coins, I only used 20 coins and now I only have 6 coins instead of 16. I would really appreciate it if I got the 10 coins that is missing :>

### Screenshots

![image](https://github.com/user-attachments/assets/602ec486-bd23-4c91-9949-1e00c2a0bc67)
![image](https://github.com/user-attachments/assets/8c5176fb-3ac8-4e7a-83a0-e548210e4c87)
![image](https://github.com/user-attachments/assets/b1483b55-3293-405f-8a79-cc5a20d73feb)
![image](https://github.com/user-attachments/assets/9e585734-c104-4d37-b392-8d525319fc08)

**Comment by xcyberwizard** [comment]
yes my coins also missing. 3x coins didnt get added to my store :(
